dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -2342,7 +2342,8 @@
       "enum": [
         "CustomSingleLabelClassification",
         "CustomMultiLabelClassification",
-        "CustomEntityRecognition"
+        "CustomEntityRecognition",
+        "CustomAbstractiveSummarization"
```

Comment:
CustomAbstractiveSummarization is not added to analyzetext. It should be in both, right?

Reply:
Good catch! Just added.